### PR TITLE
Updated go sdk version

### PR DESCRIPTION
**Pull request by dginty4**

### CODE REVIEW

File: genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go [hunk 1]
```diff
@@ -5,11 +5,11 @@ package genesyscloud
...
-// 	"github.com/google/uuid"
-// 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-// 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-// 	"github.com/mypurecloud/platform-client-sdk-go/v105/platformclientv2"
-// )
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v109/platformclientv2"
+)
```
Comment (charliecon) [inline genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go:12]: Looks like these lines should be commented out. The `import` part is commented out but these aren't

File: genesyscloud/resource_genesyscloud_knowledge_v1_document_test.go [hunk 2]
```diff
@@ -5,11 +5,11 @@ package genesyscloud
...
-// 	"github.com/google/uuid"
-// 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-// 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-// 	"github.com/mypurecloud/platform-client-sdk-go/v105/platformclientv2"
-// )
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v109/platformclientv2"
+)
```
Comment (charliecon) [inline genesyscloud/resource_genesyscloud_knowledge_v1_document_test.go:12]: Same thing here

**Comment by charliecon** [comment]
Looks to be an issue with `go generate` as well

**Review by carnellj-genesys — APPROVED** [review]

File: genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go (hunk 1 above)
Comment (charliecon) [reply genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go]: It says outdated but I think this file is still messed up - [https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/b31f8a04cbe278514bbb2848f9861608eab90a52/genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go](https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/b31f8a04cbe278514bbb2848f9861608eab90a52/genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go)

Comment (dginty4) [reply genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go]: That's weird, when I view it through the link above it shows the old file before I fixed it, but when I view it in the changes tab it is right

Comment (charliecon) [reply genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go]: My bad. I can see it's right here -[https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/NO-JIRA-Update-Go-SDK-version/genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go](https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/NO-JIRA-Update-Go-SDK-version/genesyscloud/resource_genesyscloud_knowledge_v1_category_test.go)

**Pull request merged**